larray/core.py

```diff
@@ -3770,28 +3781,33 @@ class LArray(object):
     """
     A LArray object represents a multidimensional, homogeneous
     array of fixed-size items with labeled axes.
+    It has also associated attributes (title, description, ...)
+    defined at object creation. These attributes can be passed
+    as keyword arguments at initialization.
 
     The function :func:`aslarray` can be used to convert a
     NumPy array or PandaS DataFrame into a LArray.
 
-    Parameters
-    ----------
-    data : scalar, tuple, list or NumPy ndarray
-        Input data.
-    axes : collection (tuple, list or AxisCollection) of axes \
-    (int, str or  Axis), optional
-        Axes.
-    title : str, optional
-        Title of array.
-
     Attributes
     ----------
     data : NumPy ndarray
         Data.
     axes : AxisCollection
         Axes.
-    title : str
-        Title.
+    \**kwargs :
+        Attributes (title, description, creation date, ...)
+        associated with the array
```

Comment:
Unsure this is a good idea because we have the potential for a conflict between a user-defined argument and a future argument we add. And if the user makes a typo for an existing argument it will just work (and create a user-defined attribute instead). e.g. LArray(data, axis=Axis('sex', 'H,F'))